### DISCUSSION TITLE
MY FRIEND NEEDS TO RECOVER IS ACCOUNT

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49867] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, So my friend's GitHub account https://github.com/oladetohun1 was hacked recently. His primary email and other emails connected to the account have been removed making him lose access to his account. The account is currently affiliated with many programs so it is very important to him. Please is there any way it can be recovered as soon as possible?

I have the Gmail the person used to replace my friend's primary email as well as he has updated a repo of my friend so that when a link there a clicked you unknowingly  download  a malicious software. 

https://github.com/oladetohun1/How-To-Download-IDM-For-Free-2023-Actually-Method
That is the  repo the 'hacker' has  updated. Just to be clear

**Comment by blackbirdcoder** [comment]
Perhaps his code does not fit the github policy. Contact github support
It's just guesswork

**Reply by ghost** [reply]
Someone actually got his system remotely and was able to take advantage.

**Comment by bbkx226** [comment]
If your friend had enabled two-factor authentication, they should check if they have any backup codes saved. These backup codes can be used to regain access to the account.

**Reply by ghost** [reply]
Unfortunately, he didn't

**Reply by ck-tech01** [reply]
i am too in the same situation, i had enabled two-factor authentication. what are my next steps to undertake.

**Comment by airtower-luna** [comment, marked as answer]
Please [contact GitHub support](https://support.github.com/request) ASAP, us other users can't help there. Reporting your friend's account might be another option. Either way, good luck!

**Reply by ghost** [reply]
i have done that

**Reply by ghost** [reply]
i have done that

**Comment by ERICCHW97** [comment]
Does he get back his account after contact support?
I have the same situation here :(
Hacker applied 2FA on my account and changed the password.
Also, created some fake repo.
I used SSH key to remove those commit, but cannot delete repo or take control for the account.

**Reply by iMirai007** [reply]
my account was hacked in may 29 hacker remove my email  till now github no respond im losing hope finding github alternative

**Comment by iMirai007** [comment]
update i got back my account yay but it take almost a year

**Reply by MurageMaina** [reply]
experiencing the same thing how did your account back
